charts/prometheus-postgres-exporter/templates/NOTES.txt

```diff
@@ -1,3 +1,6 @@
+#################################################################################
```

Comment:
As the chart will not work in helm 2 i guess this notifications will not be shown to a helm 2 user anyway.

Reply:
Er... @monotek I think previously @scottrigby would like to have 'crystal clear' on supporting helm3. I don't know what else / where else I could put to make it so. What would you guys suggest? cc @scottrigby

Reply:
I think it's about to put it in the upgrading section of the chart.

Reply:
Yep, this is a suggestion not a demand. But it came from trial and error in the helm/charts repo. See this note in the [review guidelines](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning) there:

> Any breaking (backwards incompatible) changes to a chart should:
> 1. Bump the MAJOR version
> 2. In the README, under a section called "Upgrading", describe the manual steps necessary to upgrade to the new (specified) MAJOR version

This came from many conversations about upgrading confusion, which lead to this issue https://github.com/helm/charts/issues/5657 and ultimately https://github.com/helm/charts/pull/11207.

Personally I'd suggest adopting the same policy for this repo too. Just to make it clear to end users what upgrading to each MAJOR version of a chart means, without them having to do some version diff of the requirements section in the chart README 😛